.changelog/2609.feature.md

```diff
@@ -0,0 +1,5 @@
+Export Ledger entities
```

Comment:
This doesn't seem ledger specific? I would rather says "Export signer public keys as empty entities" or something like that.